documentation/release_notes.markdown

```diff
@@ -271,17 +345,17 @@ derive data from a PuppetDB export tarball.
 
 * Allow factset hashes to be nil in db
 
-    This is necessary for factsets to function after the PDB-898 migration,
+    This is necessary for factsets to function after the ([PDB-898](https://tickets.puppetlabs.com/browse/PDB-898)) migration,
     since existing factsets are not being hashed.
 
-* Fix errors on PDB shutdown ([PDB-880](https://tickets.puppetlabs.com/browse/PDB-880))
+* Fix errors on PDB shutdown ([PDB-880](https://tickets.puppetlabs.com/browse/PDB-880)) ([PDB-1102](https://tickets.puppetlabs.com/browse/PDB-1102))
 
-    This patch bumps activemq to 5.7.0.  This seems to prevent two errors that
-    would sometimes occur on shutdown, one where the broker was stopped before its consumers
-    were removed and another where KahaDB batching fails to reset due to the PageFile
-    not being loaded.
+    To fix errors on PDB shutdown, we upgrade ActiveMQ to 5.7.0 (later 5.11.1). This seems to
+    prevent two errors that would sometimes occur on shutdown, one where the broker was stopped
+    before its consumers were removed and another where KahaDB batching fails to reset due to
+    the PageFile not being loaded.
```

Comment:
This should be reworded, it kind of sounds like we don't really know if it fixed things.
`ActiveMQ was upgraded to ... . This was done to prevent two errors ... .`